input2d.timestepping.all_t

```diff
@@ -0,0 +1,158 @@
+USE_PRECONDITIONER =  TRUE
+W = 0.75  // under relaxation factor for box relaxation scheme
+RHO = 1.0
+MAX_MULTIGRID_LEVELS = 3
+START_TIME = 0.0
+END_TIME = 1.0
+NUM_CYCLES = 1
+N = 32
+DX = 1.0 / N 
+U_MAX = 2.0 
+CFL = 0.5
+DT_MAX = CFL * DX / U_MAX 
+ENABLE_LOGGING = TRUE
+DIR_NAME = "viz2d_timestepping"
+VIZ_DUMP_TIME_INTERVAL = 0.01
+VISCOUS_TIME_STEPPING_TYPE = "TRAPEZOIDAL_RULE"
+
+// Inital conditions
+un {
+   function_0 = "cos(2*PI*X_0)"
+   function_1 = "cos(2*PI*X_1)"
+}
+
+us {
+   function_0 = "-cos(2*PI*X_0)"
+   function_1 = "-cos(2*PI*X_1)" 
+} 
+
+p {
+   function = "sin(2*PI*X_0)"
+} 
+
+f_un {
+   function_0 = "0.5*(2.0*PI*cos(2*PI*t)+(1+2*PI+4*PI*PI)*cos(2*PI*X_0)+sin(2*PI*t))"
+   function_1 = "0.5*(2.0*PI*cos(2*PI*t)+(1+4*PI*PI)*cos(2*PI*X_1)+sin(2*PI*t))"
+}
+
+f_us {
+   function_0 = "-PI*cos(2*PI*t)+(-0.5+PI-2*PI*PI)*cos(2*PI*X_0)-0.5*sin(2*PI*t)"
+   function_1 = "0.5*(-2*PI*cos(2*PI*t)-(1+4*PI*PI)*cos(2*PI*X_1)-sin(2*PI*t))"
+}
```

Comment:
Okay, yes there's an additional sin(2*PI*t) term due to the drag term. This was missing in my exact solution.